Content.Server/Access/Systems/IdCardSystem.cs

```diff
@@ -74,5 +80,53 @@ private void UpdateEntityName(EntityUid uid, IdCardComponent? id = null)
                                 ("fullName", id.FullName),
                                 ("jobSuffix", jobSuffix));
         }
+
+        /// <summary>
+        ///     Attempt to find an ID card on an entity. This will look in the entity itself, in the entity's hands, and
+        ///     in the entity's inventory.
+        /// </summary>
+        public bool TryFindIdCard(EntityUid uid, [NotNullWhen(true)] out IdCardComponent? idCard)
+        {
+            // check held item?
+            if (EntityManager.TryGetComponent(uid, out SharedHandsComponent? hands) &&
+                hands.TryGetActiveHeldEntity(out var heldItem) &&
+                TryGetIdCard(heldItem.Uid, out idCard))
+            {
+                return true;
+            }
+
+            // check entity itself
+            if (TryGetIdCard(uid, out idCard))
+                return true;
+
+            // check inventory slot?
+            if (EntityManager.TryGetComponent(uid, out InventoryComponent? inventoryComponent) &&
+                inventoryComponent.HasSlot(EquipmentSlotDefines.Slots.IDCARD) &&
+                inventoryComponent.TryGetSlotItem(EquipmentSlotDefines.Slots.IDCARD, out ItemComponent? item) &&
+                TryGetIdCard(item.Owner.Uid, out idCard))
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        ///     Attempt to get the id card component from an entity. This will also check if this entity has is a PDA,
```

Comment:
There's typo in the comment, it says 

> This will also check if this entity has is a PD

should remove either the **has** or **is**

Reply:
I've re-worded the whole comment, thanks for spotting that.